### PR TITLE
Add EventProcessor and change some mistakes like wrong names of class…

### PR DESCRIPTION
**Pull request by tdudzik**
…es in logger and wrong names of directories.

### CODE REVIEW

**Comment by clinton56** [comment]
Please rebase your branch

**Comment by clinton56** [comment]
I accept

File: client/src/main/java/io/github/expansionteam/battleships/logic/EventHandler.java [hunk 1]
```diff
@@ -9,15 +9,18 @@
 
 public class EventHandler {
 
-    private final static Logger log = Logger.getLogger(MainLauncher.class.getSimpleName());
+    private final static Logger log = Logger.getLogger(EventHandler.class.getSimpleName());
```
Comment (LIttleAncientForestKami) [inline client/src/main/java/io/github/expansionteam/battleships/logic/EventHandler.java]: Why the `getSimpleName` and not just `.class`?

File: client/src/main/java/io/github/expansionteam/battleships/logic/client/SocketClient.java [hunk 2]
```diff
@@ -30,7 +30,7 @@ public String talkWithServer(String str) {
             dataOutputStream.flush();
 
             message = dataInputStream.readUTF();
-            log.debug("[SERVER] Message arrived: " + message);
+            log.debug("[CLIENT] Message arrived: " + message);
 
             // TODO: change ending game - send a message from the server, when one disconnected
         } catch (EOFException e) {    // ending game
```
Comment (LIttleAncientForestKami) [inline client/src/main/java/io/github/expansionteam/battleships/logic/client/SocketClient.java]: This comment should be in the exception message. I'd also repackage the exception into `AbruptGameEnding` or something that would mean not just EOF but game connection breaking.

File: client/src/main/java/io/github/expansionteam/battleships/logic/EventHandler.java (hunk 1 above)
Comment (clinton56) [inline client/src/main/java/io/github/expansionteam/battleships/logic/EventHandler.java]: Yes, you are right. I added `getSimpleName` to exclude package names, but just `.class` works too ;)

**Pull request merged**